pkg/controllers/nodepool/counter/controller.go

```diff
@@ -89,10 +102,12 @@ func (c *Controller) resourceCountsFor(ownerLabel string, ownerName string) v1.R
 		}
 		if n.Labels()[ownerLabel] == ownerName {
 			res = resources.MergeInto(res, n.Capacity())
+			nodeCount += 1
 		}
 		return true
 	})
-	return functional.FilterMap(res, func(_ v1.ResourceName, v resource.Quantity) bool { return !v.IsZero() })
+	res[v1.ResourceName("nodes")] = resource.MustParse(fmt.Sprintf("%d", nodeCount))
```

Comment:
```suggestion
	res[ResourceNameNodes] = resource.MustParse(fmt.Sprintf("%d", nodeCount))
```

Can you codify this into some kind of constant at the top of the file?